arcaflow_plugin_stressng/stressng_schema.py

```diff
@@ -731,14 +750,15 @@ class StressNGParams:
         ),
     ] = None
 
+    taskgroup = r"(?:\d{1,3}|\d{1,3}-\d{1,3})"
     taskset: typing.Annotated[
         typing.Optional[str],
-        validation.pattern(re.compile(r"((\d{1,3})(\,|\-))*\d{1,3}")),
+        validation.pattern(re.compile(f"^{taskgroup}(?:,{taskgroup})*$")),
```

Comment:
While it is parsimonious to put the grouping in the `taskgroup` string, it looks a little weird.  (I.e., out of context, there is no reason to group an entire expression.)  Consider putting the grouping in the reference, instead:
```python
    taskgroup = r"\d{1,3}|\d{1,3}-\d{1,3}"
...
        validation.pattern(re.compile(f"^(?:{taskgroup})(?:,(?:{taskgroup}))*$")),
```

Reply:
It's exactly how you suggested it previously. :laughing:

Reply:
Yeah, well, my reviewer had a slightly different take on it, apparently.  🙃